src/applications/gi-sandbox/containers/SearchPage.jsx

```diff
@@ -87,6 +88,7 @@ export function SearchPage({
   return (
     <span className="search-page">
       <div className={searchPageClasses}>
+        <GIBillHeaderInfo />
```

Comment:
This should exist outside of any pre-existing element so as to preserve dom order